### PR TITLE
Bump article dependency version

### PR DESCRIPTION
**Pull request by fourlastor**
We need the play icon on the lead asset video button, which has been done in https://github.com/newsuk/times-components/pull/809

This bumps the dependency version so we'll be able to use that.

Another change we need is in `@times-components/provider`, so that we'll be able to query for brightcove data.

### CODE REVIEW

File: packages/article/package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
   },
   "dependencies": {
     "@thetimes/with-client": "1.0.7",
-    "@times-components/article": "0.62.2",
+    "@times-components/article": "0.62.13",
     "@times-components/provider": "0.26.0",
```
Comment (tuncaulubilge) [inline packages/article/package.json]: Might be safer to bump it all: including provider and author-profile

Comment (fourlastor) [reply packages/article/package.json]: Ah actually I need the new provider as well, cause we need to add the fields to the query, well spotted.

Comment (fourlastor) [reply packages/article/package.json]: For the author profile, that shouldn't be an issue, let's try without first.

Comment (tuncaulubilge) [reply packages/article/package.json]: Linking tends to get messed up with multiple different versions but let's give it a try.
The problem is, almost every package in times-components uses markup, and markup uses almost all packages, so they all have dependencies on each other.

**Pull request merged**